packages/opentelemetry-exporter-prometheus/src/utils.ts

```diff
@@ -0,0 +1,99 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as prom from 'prom-client';
+import { Histogram, Distribution } from '@opentelemetry/metrics';
+
+/**
+ * Opentelemetry's metrics API match closely what is done inside `prom-client`
+ * so to avoid having to compute data both inside otel and prom. We use this
+ * wrapper that looks like a internal prom implementation but is actually just
+ * using the Opentelemetry data.
+ */
+
+// Fake implementation of the prom Histogram that allows us to set its internal state
+export class FakeHistogram extends prom.Histogram {
+  // @ts-ignore
+  // tslint:disable-next-line
+  private upperBounds: number[] = [];
+  // ref https://github.com/siimon/prom-client/blob/master/lib/histogram.js#L34
+  // tslint:disable-next-line
```

Comment:
What are you disabling here? If you are going to disable a rule, please disable it specifically by name to make it obvious.